cylc/flow/scheduler_cli.py

```diff
@@ -42,67 +42,45 @@
 from cylc.flow import suite_files
 from cylc.flow.terminal import cli_function
 
-RUN_DOC = r"""cylc [control] run|start [OPTIONS] [ARGS]
+PLAY_DOC = r"""cylc [control] play [OPTIONS] [ARGS]
 
-Start a new suite run.
-
-Any dependencies prior to the start point will be ignored.
-
-WARNING: this will wipe out previous suite state. To restart from a previous
-state, see 'cylc restart --help'.
+Start running a workflow, or restart a stopped workflow from its previous
+state/cycle point, or resume a paused workflow by releasing all tasks.
 
 The scheduler will run as a daemon unless you specify --no-detach.
 
-If the suite is not already installed (by "cylc install" or a previous run)
+If the workflow is not already installed (by "cylc install" or a previous run)
 it will be installed on the fly before start up.
 
 Examples:
-    # Run the suite installed with name REG.
-    $ cylc run REG
-
-    # Install $PWD/flow.cylc as $(basename $PWD) and run it.
-    # Note REG must be given explicitly if START_POINT is on the command line.
-    $ cylc run
-
-A "cold start" (the default) starts from the suite initial cycle point
-(specified in flow.cylc or on the command line). Any dependence on tasks
-prior to the suite initial cycle point is ignored.
+    # Start/restart the workflow with name REG.
+    $ cylc play REG
 
-A "warm start" (-w/--warm) starts from a given cycle point later than the suite
-initial cycle point (specified in flow.cylc). Any dependence on tasks prior
-to the given warm start cycle point is ignored. The suite initial cycle point
-is preserved."""
+A "cold start" (the default for a freshly-installed workflow) starts from the
+initial cycle point (specified in flow.cylc or on the command line). Any
+dependence on tasks prior to the initial cycle point is ignored.
+It is also possible to start from a point that is later than the initial cycle
+point, using the option --startcp=CYCLE_POINT. The initial cycle point is
+preserved, but the workflow does not start there and instead starts part-way
+through the graph (historically known as a "warm start").
 
-RESTART_DOC = r"""cylc [control] restart [OPTIONS] ARGS
+A "restart" continues the workflow from the earliest cycle point that was
+incomplete when shutdown occurred. Tasks recorded as submitted or running are
```

Comment:
```suggestion
A "restart" continues on from the most recent recorded state of the workflow. 
Tasks recorded as submitted or running are
```
I know what you mean, but "continues from the [blah] cycle point ..." could be confusing.  We've traditionally described warm starts that way. A restart often involves tasks at multiple cycle points.